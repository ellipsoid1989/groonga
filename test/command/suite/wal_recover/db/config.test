# _database_close/_database_reopen can't use with HTTP because requests via HTTP are always
# processed by workers. Workers never use primary WAL role.
#@require-interface stdio

#@require-feature message_pack

#$GROONGA_ALLOW_DATABASE_REOPEN=yes
#$GRN_WAL_ROLE=primary

#@copy-path #{db_path}.conf #{db_path}.conf.initial

config_set pgroonga_writable true

dump

#@copy-path #{db_path}.conf.wal #{db_path}.conf.wal.keep
_database_close

#@copy-path #{db_path}.conf.initial #{db_path}.conf
#@copy-path #{db_path}.conf.wal.keep #{db_path}.conf.wal
_database_reopen

dump
