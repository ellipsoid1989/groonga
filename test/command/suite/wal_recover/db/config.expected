config_set pgroonga_writable true
[[0,0.0,0.0],true]
dump
config_set pgroonga_writable true
_database_close
[[0,0.0,0.0],true]
_database_reopen
[[0,0.0,0.0],true]
dump
config_set pgroonga_writable true
